src/OpenTelemetry.Exporter.Geneva/GenevaLogExporter.cs

```diff
@@ -31,6 +34,25 @@ public GenevaLogExporter(GenevaExporterOptions options)
 
         bool useMsgPackExporter;
         var connectionStringBuilder = new ConnectionStringBuilder(options.ConnectionString);
+
+        if (connectionStringBuilder.PrivatePreviewEnableUserEvents)
+        {
+#if NET
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+            {
+                throw new ArgumentException("Exporting data in user_events is only supported for .NET 8 or later on Linux.");
```

Comment:
nit: I think the exception messages here could be more specific to avoid confusion.

This exception at line 43 is only due to Linux.
The second exception at line 52 seem specific to NetFX.

Reply:
If we could detect the runtime version, that would help provide detailed error messages.

Reply:
I was leaving the ambiguity on purpose. No matter the customer is using .NET Framework or older .NET, we can error with the same message. They would know what version they're running anyways.